fe2-android/app/src/main/res/layout/layout_election_display.xml

```diff
@@ -59,8 +59,6 @@
             android:id="@+id/election_action_button"
             android:layout_width="wrap_content"
             android:layout_height="@dimen/event_buttons_height"
-            android:text="@{eventCategory == EventCategory.PRESENT ? `Cast Vote ` : `Tally up Votes `}"
-            android:enabled="@{eventCategory == EventCategory.PRESENT ? true : view_model.isOrganizer() ? true : false}"
```

Comment:
This election action button doesn't have a text anymore and is always enabled, is this the desired action?

Reply:
Yes, all of that is now set in the setupElectionElement method in EventExpandableListViewAdapter. I know that it is better to have layout-related code directly in the xml, but here, the text changes when an event happens (hence requiring use of an observer). If the text was already defined in the xml, when clicking on another layout categroy (e.g PRESENT, whereas the election is in PAST), this would reload the xml, and "override" the text. With the old version, that would then change the text from "Waiting for results" to "Tally up votes". As for enabling, it seemed easier to enable/disable depending on the category and the event in the method as well.